src/MIDebugEngine/Natvis.Impl/Natvis.cs

```diff
@@ -61,6 +62,15 @@ public void Dispose()
         {
         }
         public bool IsPreformatted { get { return Parent.IsPreformatted; } set { } }
+
+        public string Address()
+        {
+            throw new NotImplementedException();
```

Comment:
>throw new NotImplementedException(); [](start = 12, length = 36)

I think you instead want to call the Parent for these two.

Reply:
`public string Address() { return Parent.Address(); }` and `public string Size() { return Parent.Size(); }` break Break When Value Changes. I can look into why this occurs.